docs/Quickstart_wuf.md

```diff
@@ -0,0 +1,145 @@
+
+To enroll a new user:
+
+1. Make sure your local service is running, or start it by using this command, if it's not.
+
+    ```shell
+    cd <your-github-workspace>/to-do-service/api 
+    sh start-server.sh
+    ```
+
+2. Open another IDE.
+3. Use the POST command to send new request with these values:
+
+    ```shell
+    curl -X POST http://localhost:3000/users \
+    > -H "Content-Type: application/json" \
+    > -d '{"last_name": "Fish", "first_name": "Wendy", "email": "wfish@example.com"}'
+    ```
+
+4. The response body should look something like this with the user name you supplied. The response should also include a new user `id`.
+
+    ```js
+    {
+        "last_name": "Fish",
+        "first_name": "Wendy",
+        "email": "wfish@example.com",
+        "id": 7
+    }
+    ```
+
+### Add tasks
+
+Now we are going to add some tasks for our new user. You'll note that to do this we use the same `POST` command that we used to create the `user` record. The difference is that now we need to supply the following task data.
+
+| Property name | Type | Description |
+| ------------- | ----------- | ----------- |
+| `user_id` | number | The ID of the user resource to which this task is assigned |
+| `title` | string | The title or short description of the task |
+| `description` | string | The long description of the task|
+| `due_date` | string | The [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format of the date and time the task is due |
+| `warning` | number | The number of minutes relative to the `due_date` to alert the user of the task. This is normally a negative number to alert the user before the `due_date`.|
+
+
+
+1. We'll use the `POST` command to create a task for our new user to **get plane tickets from NYC to Portugal** before midnight on **11/01/24**. We will also send a reminder **24 hours** before the deadline.
+
+    ```shell
+        curl -X POST http://localhost:3000/tasks \
+        > -H "Content-Type: application/json" \
+        > -d '{"user_id": 7, "title": "Get plane tickets", "description": "Find a flight from NYC to Lisbon, Porto, or Sao Miguel", "due_date": "2024-11-01T23:59", "warning": "-1440"}'
+    ```
+2. The response should look something like this with a new task `id`.
+    ```shell
+    {
+    "user_id": 7,
+    "title": "Get plane tickets",
+    "description": "Find a flight from NYC to Lisbon, Porto, or Sao Miguel",
+    "due_date": "2024-11-01T23:59",
+    "warning": "-1440",
+    "id": 8
+    }
+    ```
+1. Create some more tasks for your new user.
+
+### Get the tasks for a user
+Now we'll use the service to get all the tasks we created for our new user. This is a good check to see one way this API is used.
+1. Add the `user_id` that was returned in the [first section](#enroll-a-new-user) of this tutorial to a cURL request to the `tasks` resource.
```

Comment:
```suggestion
1. For this step, you need the `id` of the user you [enrolled earlier](#enroll-a-new-user).
1. Add the value of the new user's `id` as the value of the `user_id` parameter. In this example, the value of the  `user_id` is `7`.
```

Spelled out in more detail to help the reader associate the `user_id` to the `id` returned above.